src/interfaces/UserInterface.ts

```diff
@@ -1 +1,9 @@
-export {}
+import mongoose from 'mongoose'
+
+export default interface UserInterface {
```

Comment:
This should be a named export, we are gonna have many interfaces exported from this file.

Reply:
@nadasuhailAyesh12